Fix error
--
./racurses.c: In function ‘argus_command_string’:
./racurses.c:4654:25: error: assignment to ‘int (**)(struct ArgusRecordStruct *, struct ArgusRecordStruct *)’ from incompatible pointer type ‘int (**)(void)’ [-Wincompatible-pointer-types]
 4654 |             if ((srtalg = (int (**)()) ArgusCalloc(ARGUS_MAX_SORT_ALG, sizeof(void *))) == NULL)
--

--- a/examples/ratop/racurses.c	2025-08-09 02:23:24.726290343 +0300
+++ b/examples/ratop/racurses.c	2025-08-09 02:25:27.054693713 +0300
@@ -4651,7 +4651,7 @@
 
             if ((strbuf = ArgusCalloc(1, MAXSTRLEN)) == NULL)
                ArgusLog (LOG_ERR, "argus_command_string: ArgusCalloc error\n");
-            if ((srtalg = (int (**)()) ArgusCalloc(ARGUS_MAX_SORT_ALG, sizeof(void *))) == NULL)
+            if ((srtalg = (int (**)(struct ArgusRecordStruct *, struct ArgusRecordStruct *)) ArgusCalloc(ARGUS_MAX_SORT_ALG, sizeof(void *))) == NULL)
                ArgusLog (LOG_ERR, "argus_command_string: ArgusCalloc error\n");
 
             ptr = strbuf;
